tools/js-sdk-release-tools/src/common/utils.ts

```diff
@@ -73,4 +93,20 @@ export function tryReadNpmPackageChangelog(packageFolderPath: string): string {
         logger.logWarn(`Failed to read NPM package's changelog "${changelogPath}": ${(err as Error)?.stack ?? err}`);
         return '';
     }
-}
\ No newline at end of file
+}
+
+// generated path is in posix format
+// e.g. sdk/mongocluster/arm-mongocluster
+export async function getGeneratedPackageDirectory(typeSpecDirectory: string): Promise<string> {
+    const tspConfig = await loadTspConfig(typeSpecDirectory);
+    const serviceDir = tspConfig.parameters?.['service-dir']?.default;
+    if (!serviceDir) {
+        throw new Error(`Misses service-dir in parameters section of tspconfig.yaml. ${messageToTspConfigSample}`);
+    }
+    const packageDir = tspConfig.options?.[emitterName]?.['package-dir'];
+    if (!packageDir) {
+        throw new Error(`Misses package-dir in ${emitterName} options of tspconfig.yaml. ${messageToTspConfigSample}`);
+    }
+    const packageDirFromRoot = posix.join(serviceDir, packageDir);
```

Comment:
just curious why posix join? what if this command runs in windows environment?

Reply:
it's used in config/package that only accept posix format path.